lib/devicetrust/enroll/enroll_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package enroll_test
```

Comment:
What's the advantage of using a _test package here?

Reply:
I use `_test` packages by default, as it's a clear signal that we only test via public APIs. I tend to ask myself the opposite question, "why not a _test package?".

Reply:
Agreed, we really should default to _test.